### ISSUE TITLE
conf/runners filenames do not reflect names of scripts

### ISSUE THREAD

**Issue by raszuk** [bugy/script-server#614]
When script name is changed via admin panel conf/runners/*.json file is not renamed to reflect it. 

The workaround could be the ability to copy scripts and delete old via admin panel however it does not seem like copy and delete operations are supported. 

Ability to copy scripts in admin panel then edit just few fields in the new copy (with a new name) would be very helpful irrespective of the above. 

Thx !

**Comment by bugy** [comment]
Hi @raszuk , thanks for reporting. Regarding the filename, this one is a duplicate: https://github.com/bugy/script-server/issues/250
May be you can update the ticket, to be only about "copy" feature?

**Comment by raszuk** [comment]
@bugy Yes since the filename difference has been reported in #250 let's use this ticket to only provide ability to copy (with new name) and delete json files via admin panel.

**Comment by bugy** [comment]
I guess only copy 😅 there should be a ticket for deletion already

**Comment by raszuk** [comment]
Ok :)
